localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -914,7 +914,7 @@ msgid ""
 "Advanced options cannot be changed for the selected font.\n"
 "Select another font."
 msgstr ""
-"Las opciones avanzadas no pueden modificarse para la fuente seleccionada. "
+"Las opciones avanzadas no pueden modificarse para la fuente seleccionada. \n"
```

Comment:
No need to have a space before the line jump

Reply:
Done.